src/app/components/full-screen-button.tsx

```diff
@@ -6,24 +6,29 @@ import { ExpandIcon } from '@leather.io/ui';
 
 import { analytics } from '@shared/utils/analytics';
 
+import { whenPageMode } from '@app/common/utils';
 import { openIndexPageInNewTab } from '@app/common/utils/open-in-new-tab';
 
 export function FullScreenButton() {
   const location = useLocation();
-  return (
-    <Box
-      _hover={{ bg: 'ink.component-background-hover' }}
-      _focus={{ outline: 'none' }}
-      p="space.02"
-      onClick={() => {
-        void analytics.untypedTrack('click_open_in_new_tab', {
-          location: 'header',
-        });
-        void analytics.identify(undefined, { hasVisitedFullPageMode: true });
-        openIndexPageInNewTab(location.pathname);
-      }}
-    >
-      <ExpandIcon />
-    </Box>
-  );
+
+  return whenPageMode({
+    full: null,
+    popup: (
+      <Box
```

Comment:
I didn't pay enough attention in the previous PR, thought this was an dev-only button for some reason. Can we use an `IconButton` perhaps, with `aria-label`, and `_focusVisible` instead of `_focus`?

Reply:
Yeah good point, the only reason I did it this way was to get the hover styles the same as the menu button, let me take a look